code/game/jobs/job/medical.dm

```diff
@@ -98,13 +98,13 @@
 		H.equip_to_slot_or_del(new /obj/item/clothing/mask/surgical(H), slot_wear_mask)
 		H.equip_to_slot_or_del(new /obj/item/device/pda/viro(H), slot_belt)
 	else if(has_alt_title(H, alt_title,"Nurse"))
-		if(H.gender == FEMALE)
-			if(prob(50))
-				H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/nursesuit(H), slot_w_uniform)
-			else
-				H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/nurse(H), slot_w_uniform)
-			H.equip_to_slot_or_del(new /obj/item/clothing/head/nursehat(H), slot_head)
-		else
+		//if(H.gender == FEMALE)
+		//	if(prob(50))
+		//		H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/nursesuit(H), slot_w_uniform)
+		//	else
+		//		H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/nurse(H), slot_w_uniform)
+		//	H.equip_to_slot_or_del(new /obj/item/clothing/head/nursehat(H), slot_head)
+		//else
 			H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/medical/purple(H), slot_w_uniform)

```

Comment:
That looks like a issue with inconsistent indentation.

Just delete the block above.
Or is there a need to keep it commented out in the codebase ?

Reply:
I think that if spriters will re-sprite nurse's outfit to be full body coverage in future, this can be placed back

Reply:
Then that code can be readded.
However dead code should generally be avoided.